calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -676,7 +737,25 @@ def start(
         )
 
         if self.operation_mode == OperationMode.EVALUATION:
-            self.calibration_button.setEnabled(False)
+            #  Initial state of the elements on evaluation mode
+            self.calibrated_camera_model = self.current_camera_model
+            self.image_view_type_combobox.setEnabled(True)
+            self.undistortion_alpha_spinbox.setEnabled(True)
+            self.draw_evaluation_heatmap_checkbox.setEnabled(False)
+            self.draw_evaluation_points_checkbox.setEnabled(False)
+            self.draw_training_points_checkbox.setEnabled(False)
+            self.draw_training_heatmap_checkbox.setEnabled(False)
+            self.training_sample_slider.setEnabled(False)
+            self.evaluation_sample_slider.setEnabled(False)
+            self.image_view_type_combobox.clear()
+            # Order of of how items are added to the combobox matters,
```

Comment:
of of

Reply:
thanks i will fix it

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e